backend/db/utils.py

```diff
@@ -120,7 +120,7 @@ def current_datetime(time_zone=['UTC/GMT', 'EST/EDT'][1]) -> str:
 def last_refresh_timestamp(con: Connection) -> str:
     """Get the timestamp of the last database refresh"""
     return sql_query(
-        con, f"SELECT value FROM public.manage WHERE key = 'last_refreshed_DB';", return_with_keys=False)[0][0]
+        con, f"SELECT value FROM public.manage WHERE key = 'last_refresh_success';", return_with_keys=False)[0][0]
```

Comment:
@hugwuoke `last_refreshed_DB ` 1/2: This wasn't something I thought of during our meeting, nor is it something that our unit test(s) might have caught. I'm going to be deleting this `last_refreshed_DB` soon now that we no longer need it.